e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -93,6 +93,10 @@ describe("scenarios > dashboard", () => {
       H.queryBuilderHeader().findByText("Save").click();
       cy.findByTestId("save-question-modal").within(modal => {
         cy.findByLabelText("Name").clear().type(newQuestionName);
+        cy.findByLabelText("Where do you want to save this?").should(
```

Comment:
This is a bit of a weird test... It's been updated to say that we should suggest saving the new question in the dashboard that we started from, and we now assert that. But we never actually check that it was made there (though other tests cover this), only that it doesn't show up in the collections view. Maybe that's enough for a smoke test though